spec/models/categorization_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+# require_relative "../app/models/truck"
+# require_relative "../app/models/category"
+# require_relative "../app/models/categorization"
```

Comment:
As a best practice, I prefer to avoid merging commented-out code such as this into the `master` branch.